src/store/migrations/21_enable_solana_chain.ts

```diff
@@ -0,0 +1,69 @@
+import { ChainId, chains } from '@liquality/cryptoassets';
+import { accountCreator, getNextAccountColor } from '../../utils/accounts';
+import { getDerivationPath } from '../../utils/derivationPath';
+import { Networks } from '../../utils/networks';
+import { AccountType } from '../types';
+
+export const enableSolanaChain = {
+  version: 21,
+  migrate: async (state: any) => {
+    const accounts: any = {};
+    const enabledChains: any = {};
+    for (const walletId in state.accounts) {
+      accounts[walletId] = {};
+      enabledChains[walletId] = {};
+
+      for (const network of Networks) {
+        const accountExistsAndProperlyInitialized = state.accounts[walletId][network].find(
```

Comment:
Why would this ever be the case? The migration only runs from a version that solana was not added.

Reply:
I have just copy and paste Avalanche migration and adjust for Solana :D

Reply:
Resolved 🚀